### ISSUE TITLE
index.php

### ISSUE THREAD

**Issue by RavokTHC** [RavokTHC/sms.github.io#1]
<!doctype html>

 <html class="no-js" lang="en"> <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <title>Enviar SMS Grátis - A forma mais fácil de Enviar SMS Grátis</title>
  <meta name="description" content="Envie SMS grátis agora para seus amigos e familiares. Mande sms para qualquer celular Claro, Vivo, Tim, Oi e muito mais totalmente grátis.">
  <meta name="google-site-verification" content="kPQfA1UAefu2xFrS_wFiJc1ou-MPQWGlpAAonH03XbA" />
  <meta name="viewport" content="width=device-width; initial-scale=1.0;" />
  <link rel="stylesheet" href="boilerplate.css">
  <link rel="stylesheet" href="style.css">

  <script src="js/libs/modernizr-2.5.3.min.js"></script>
</head>
<body>

<div id="fb-root"></div>
<div id="wrapper">
	<div id="header">
		<div id="header-logo">
			<a href="index.php" title="Clique para voltar à página inicial"><img src="img/header-logo.png" alt="Enviar SMS" /></a>
		</div>
	</div>
	<div id="content">
		<div id="content-header">
			
			<script type="text/javascript">
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
		</div>
		<div id="sidebar">
			<div class="sidebar-google">
				
				<script type="text/javascript">
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
			</div>
			<div class="sidebar-google">
				
				<script type="text/javascript">
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
			</div>
			
			<ul class="social">
				<li>
					
					<iframe src="//www.facebook.com/plugins/like.php?href=http%3A%2F%2Fenviarsms.com.br%2F&amp;send=false&amp;layout=button_count&amp;width=120&amp;show_faces=false&amp;action=like&amp;colorscheme=light&amp;font=lucida+grande&amp;height=21" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:120px; height:21px;" allowTransparency="true"></iframe>
				</li>
				<li>
					<a name="fb_share" type="button" share_url="http://enviarsms.com.br"></a>
				</li>
			</ul>
			<ul class="social">
				<li>	
					<a href="https://twitter.com/share" class="twitter-share-button"
					data-url="http://enviarsms.com.br" 
					data-via="forcom" 
					data-text="Enviar SMS Grátis para qualquer celular !" 
					data-count="horizontal" 
					data-lang="en">Tweet</a>
				</li>
				<li>
					
					<div class="g-plusone" data-size="medium" data-href="http://google.com.br"></div>
				</li>
				<li>
					<div class="g-plus" data-action="share" data-annotation="bubble"></div>
				</li>
			</ul>
		</div>	
		
<div id="conteudo">
			<h1>Enviar SMS Grátis</h1>
			<img src="img/operadoras.png" alt="Operadoras para Envio de SMS Grátis" style="margin:20px auto;">
						
			<script type="text/javascript">
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
			<form method="post" action="">
				<h2>Informações do Destinatário:</h2>
				<input type="text" id="ddd_destino" name="ddd_destino" required="required" maxlength="2" placeholder="DDD" class="s1">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<input type="text" id="fone_destino" name="fone_destino" required="required" maxlength="8" placeholder="Número Destino" class="s2">
				<label for="mensagem">Mensagem (Máximo <small id="countdown"></small> caracteres)</label>
				<textarea id="mensagem" name="mensagem" required="required" rows="3" class="s3" data-maxsize="116" wrap="virtual"></textarea>
				<h2>Suas Informações:</h2>
				<input type="text" id="nome" name="nome" required="required" maxlength="250" placeholder="Seu Nome" class="s3">
				<input type="text" id="ddd_origem" name="ddd_origem" required="required" maxlength="2" placeholder="DDD" class="s1">
				<input type="text" id="fone_origem" name="fone_origem" required="required" maxlength="8" placeholder="Seu Número" class="s2">
				<input type="image" src="img/enviar-torpedo.png">
			</form>
		</div>
        
	</div>
</div><div id="container-footer">
	<div id="footer">
		<div id="footer-logo">
		</div>
		<nav id="footer-menu">
		</nav>
		<div id="footer-desc">
			<p>A forma mais fácil de Enviar SMS Grátis !</p>
		</div>
		<div id="forcom">
			<a href="http://www.forcom.com.br" target="_blank"><img src="img/forcom.png"></a>
		</div>
	</div>
</div>

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
<script>window.jQuery || document.write('<script src="js/libs/jquery-1.7.1.min.js"><\/script>')</script>

<script src="js/libs/jquery.tools.min.js"></script>
<script src="js/plugins.js"></script>
<script src="js/script.js"></script>

<script>
var _gaq=[['_setAccount','UA-27971220-15'],['_trackPageview']];
(function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
g.src=('https:'==location.protocol?'//ssl':'//www')+'.google-analytics.com/ga.js';
s.parentNode.insertBefore(g,s)}(document,'script'));
</script>
<script>
window.___gcfg = {lang: 'pt-BR', parsetags: 'onload'};
(function(doc, script) {
    var js, 
        fjs = doc.getElementsByTagName(script)[0],
        frag = doc.createDocumentFragment(),
        add = function(url, id) {
            if (doc.getElementById(id)) {return;}
            js = doc.createElement(script);
            js.src = url;
            id && (js.id = id);
            frag.appendChild( js );
        };
    // Google Analytics
    //add(('https:' == location.protocol ? '//ssl' : '//www') + '.google-analytics.com/ga.js', 'ga');
    // Google+ button
    add('https://apis.google.com/js/plusone.js');
    // Facebook SDK
	//add('https://connect.facebook.net/pt_BR/all.js#xfbml=1');
	add('//static.ak.fbcdn.net/connect.php/js/FB.Share');
    // Twitter SDK
    add('//platform.twitter.com/widgets.js');

    fjs.parentNode.insertBefore(frag, fjs);
}(document, 'script'));
</script>
</body>
</html>
